dev/ToastNotifications/ToastNotificationUtility.h

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+#include "pch.h"
+#include <winrt/Windows.Foundation.h>
+#include <ToastActivatedEventArgs.h>
+
+const std::wstring appIdentifierPath = L"Software\\Classes\\AppUserModelId\\";
+const std::wstring clsIdPath = L"Software\\Classes\\CLSID\\";
+
+
+winrt::event<winrt::Windows::Foundation::EventHandler<winrt::Microsoft::Windows::ToastNotifications::ToastActivatedEventArgs>>& GetToastHandlers();
+int& GetToastHandleCount();
```

Comment:
Why not just return an int?

```suggestion
int GetToastHandleCount();
```